packages/cli/templates/build.ts

```diff
@@ -101,7 +101,7 @@ writeFileSync(TEMPLATES_JSON_PATH, content, "utf-8");
   Building route template
 */
 const routeTemplateFile = readFileSync(
-  new URL("./route-template.ts", import.meta.url).pathname,
+  new URL("./route-template.tsx", import.meta.url).pathname,
```

Comment:
readFileSync accepts url, no need to extract pathname

Reply:
Need to up my node game 😅